### PR TITLE
Contract upgrade to fix message for ECDSA public key

### PR DESCRIPTION
**Pull request by niklabh**

### CODE REVIEW

**Review by Maar-io — CHANGES_REQUESTED** [review]
please update contract version as well.
Before we merge this it would be good to tag with version v1 and network name to keep track what was deployed on the Astar/Shiden/Shibuya

**Comment by Maar-io** [comment]
@niklabh please make PR against main branch

**Comment by niklabh** [comment]
First this PR needs to be merged https://github.com/AstarNetwork/astarbase/pull/23

**Comment by Maar-io** [comment]
@niklabh all contract changes need to be for main branch as well

File: contract/contracts/AstarBaseV3.sol [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/CountersUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "./DappsStakingDummy.sol";
+import "./SR25519Dummy.sol";
+import "./ECDSADummy.sol";
+
+/// @author The Astar Network Team
+/// @title Astarbase. A voluntary mapping of accounts ss58 <> H160
+contract AstarBaseV3 is Initializable, OwnableUpgradeable {
```
Comment (Maar-io) [inline contract/contracts/AstarBaseV3.sol:13]: it is V2

**Review by Maar-io — CHANGES_REQUESTED** [review]
Please first rebase contract on main and than add your changes

**Pull request merged**